docs/source/releases/v1_13_0a0.rst

```diff
@@ -17,6 +17,34 @@ Version 1.13.0a0 (2024-04-24)
 
   * Update update_this_release_note
   * Add v1_13_0a0.rst release note
+* Enhancements
+
+  * Added 'expose' command
+
+Enhancements
+============
+
+Added 'expose' command
+----------------------
+
+*From GEOIPS#274: 2023-08-19, Add useage to setup.sh for command discovery*
+
+The aforementioned issue requested that we add functionality to expose all commands
```

Comment:
can you clarify *what* you're exposing them to (eg. the CLI)

Reply:
Sure. Currently I'm exposing them to the terminal for a user to be aware of what commands are available. IMO, this is something that I'd expect would be added to the CLI in the future.

Reply:
Got it - thanks for the extra info, that makes this PR more intelligible :)

Reply:
Updated this to talk about that.